website/content/docs/release-notes/consul/v1_16_x.mdx

```diff
@@ -0,0 +1,56 @@
+---
+layout: docs
+page_title: 1.16.x
+description: >-
+  Consul release notes for version 1.16.x
+---
+
+# Consul 1.16.0
+
```

Comment:
```suggestion
We are pleased to announce the following Consul updates. 
```
Just as a matter of style, we're trying to make sure that subtopics have some kind of introduction so that we don't have headings stacked on top of each other.